core/js/src/main/scala/org/http4s/nodejs/ClientRequest.scala

```diff
@@ -28,7 +28,7 @@ import scala.scalajs.js
 /** Facade for [[https://nodejs.org/api/http.html#class-httpclientrequest]]
   */
 @js.native
-trait ClientRequest extends js.Object with Writable {
+private[http4s] trait ClientRequest extends js.Object with Writable {
```

Comment:
IIUC, that change leads to the failed CI https://github.com/http4s/http4s/actions/runs/10777879257/job/29887975208?pr=7518#step:18:53
But 0.23 lacks this private modifier — https://github.com/http4s/http4s/blob/series/0.23/core/js/src/main/scala/org/http4s/nodejs/ClientRequest.scala#L33
Where does from it come?

Reply:
I saw it as new in `main`, but I had to do this merge twice because I neglected to pull main before the first merge.  It's possible I accidentally reintroduced something that had come and gone from `main`.